### PR TITLE
This PR makes explicit how to use email channel for Verify

### PR DESCRIPTION
**Pull request by leandroshp**
This PR makes explicit how to use email channel for Verify. It adds a method to retrieve a Email Message and also adds a new example on how to use Verify with email

### CODE REVIEW

File: lib/messagebird.js [hunk 1]
```diff
@@ -475,6 +475,29 @@ module.exports = function (accessKey, timeout, features) {
...
+      /**
+       * Send a verification code via email
+       *
+       * @param {String} from
+       * @param {String} to
+       * @param {Object} params
+       * @param {Function} callback
+       * @return {void}
+       */
+       createWithEmail: function (from, to, params, callback) {
```
Comment (aodinok) [inline lib/messagebird.js:487]: Can you please also add new methods to TypeScript typings in index.d.ts?

File: lib/messagebird.js [hunk 2]
```diff
@@ -500,6 +523,17 @@ module.exports = function (accessKey, timeout, features) {
...
+      },
+
+      /**
+       * Get email message details
+       *
+       * @param {String} email message id
+       * @param {Function} callback
+       * @return {void}
+       */
+      getVerifyEmailMessage: function (id, callback) {
```
Comment (aodinok) [inline lib/messagebird.js:535]: Also would be really nice to add simple tests for new methods 🙏

**Review by aodinok — APPROVED** [review]
Please also run `npm run dtslint` to make sure TS typings that will be added are ok

File: lib/messagebird.js (hunk 2 above)
Comment (leandroshp) [reply lib/messagebird.js:535]: I have added the tests

File: lib/messagebird.js (hunk 1 above)
Comment (leandroshp) [reply lib/messagebird.js:487]: fixed!

**Pull request merged**